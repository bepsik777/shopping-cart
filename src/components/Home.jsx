
const Home = () => {
    return (
        <div className="text-3xl bg-white flex grow justify-center items-center selection:bg-sky-600 selection:text-white">I am Home</div>
    )
}

export default Home