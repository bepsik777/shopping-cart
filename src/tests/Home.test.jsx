import { describe, it, expect } from 'vitest';
import { render, screen } from '@testing-library/react';

import Home from '../components/Home';

describe("App component", () => {
  it('render home', () => {
    render(<Home/>)
    const homeText = screen.getByText("I am Home")
    expect(homeText).toMatchSnapshot()
  })
})

