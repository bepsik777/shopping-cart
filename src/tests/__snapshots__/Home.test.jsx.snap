// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App component > render home 1`] = `
<div
  class="flex grow justify-center items-center"
>
  I am Home
</div>
`;
